### DISCUSSION TITLE
Could we implement spatiotemporal Kriging using this Pykrige package?

### DISCUSSION THREAD

**Discussion by GIS-PanTianLi** [GeoStat-Framework/PyKrige#222] [Q&A]
There is KrigeST in R language, and I wonder whether we could do it in this python package?

**Comment by MuellerSeb** [comment]
That is at least the idea for the future. For now we have to stick with the metric spatio-temporal model, that uses one direction as time dimension. In GSTools, this is already explained:
https://geostat-framework.readthedocs.io/projects/gstools/en/stable/examples/09_spatio_temporal/index.html
